app/controllers/lessons_controller.rb

```diff
@@ -82,6 +85,11 @@ def set_video_link
     @lesson.save!
   end
 
+  def lesson_image_params
```

Comment:
It's better to name this method set_lesson_image_params

Reply:
Ok.